backends/graphics/opengl/opengl-graphics.cpp

```diff
@@ -1633,9 +1633,10 @@ bool OpenGLGraphicsManager::gameNeedsAspectRatioCorrection() const {
 		const uint height = getHeight();
 
 		// In case we enable aspect ratio correction we force a 4/3 ratio.
-		// But just for 320x200, 640x400 and Hercules games, since other
+		// But just for 320x200, 640x400, 640x350 (16 color VGA) and Hercules games, since other
```

Comment:
Nitpick: 640 x 350 is an EGA resolution. Dark Seed uses a trick to use the VGA palette in this mode, but that is not normally available.

Reply:
Good point I'll change the text to clarify that